client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -34,26 +33,22 @@ import {
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
+// Maps a field in the settings form to a checklist task slug
+const FIELDS_TO_LAUNCHPAD_TASKS = {
+	blogname: 'site_title',
+	'subscription_options.welcome': 'customize_welcome_message',
+};
+
 const debug = debugFactory( 'calypso:site-settings' );
 
 const wrapSettingsForm = ( getFormSettings ) => ( SettingsForm ) => {
 	class WrappedSettingsForm extends Component {
 		state = {
 			uniqueEvents: {},
-			isSiteTitleTaskCompleted: false,
-			blogNameChanged: false,
 		};
 
 		componentDidMount() {
 			this.props.replaceFields( getFormSettings( this.props.settings, this.props ) );
-
-			// Check if site_title task is completed
-			fetchLaunchpad( this.props.siteSlug, 'intent-build' ).then( ( { checklist_statuses } ) => {
```

Comment:
Switching from this explicit `fetchLaunchpad` to `withCompleteLaunchpadTasksWithNotice` means that we're no longer passing the explicit `intent-build` checklist slug.
That's probably a good thing, it's weird to pass `intent-build` when the site doesn't necessarily have that intent.
Just though I'd point it out.

Reply:
In my testing the `intent-build` flow still works as expected, and there aren't any active flows where the site name task is actually done as part of the launchpad (the site title has usually been chosen by the time the user reaches My Home).

So we're not at any risk of breaking things here.